### PR TITLE
Merchant item show

### PR DESCRIPTION
**Pull request by rcasias**
Hey Everyone! let me know what you think and would like me to update! Thank you!

### CODE REVIEW

File: app/models/merchant.rb [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  # end
+
+  def top_5
+    items.joins(:transactions)
+      .select('items.id', 'sum(invoice_items.quantity * invoice_items.unit_price) as revenue', 'transactions.result', 'items.name')
+      .where("transactions.result = 'success'")
+      .group('items.id')
+      .group('transactions.result')
+      .order('revenue DESC')
+      .limit(5)
```
Comment (Caleb1991) [inline app/models/merchant.rb:18]: Fantastic use of grouping here, a real great showing of ActiveRecord as well!

File: app/models/merchant.rb [hunk 2]
```diff
@@ -1,3 +1,23 @@
 class Merchant < ApplicationRecord
   has_many :items
+  has_many :invoice_items, through: :items
+
```
Comment (Caleb1991) [inline app/models/merchant.rb:6]: Good use of relationships here!

File: app/views/merchant_items/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+   <%= form.label :unit_price %><br/>
+   <%= form.text_field :unit_price %><br/>
+
+   <%= form.label :merchant_id, style: "display:none;"  %><br/>
+   <%= form.text_field :merchant_id, value: "#{@merchant.id}", style: "display:none;" %><br/>
+
+   <%= form.label :status, style: "display:none;"  %><br/>
+   <%= form.text_field :status, value: "disable", style: "display:none;" %><br/>
+
```
Comment (Caleb1991) [inline app/views/merchant_items/new.html.erb:17]: Such a cool way of passing defaults, I had never known this way of doing this!

File: spec/features/merchant_items/index_spec.rb [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  # I see a list of the names of all of my items
+  # And I do not see items for any other merchant
+  it 'Can can visist a merchant index page' do
+    merchant = Merchant.create!(name: 'Schroeder-Jerde')
+    merchant.items.create!(name: 'Item Qui Esse', description: 'description', unit_price: 7510)
+    visit "/merchants/#{merchant.id}/items"
+    expect(current_path).to eq("/merchants/#{merchant.id}/items")
+    expect(page).to have_content('Schroeder-Jerde')
+    expect(page).to have_content('Item Qui Esse')
+  end
```
Comment (Caleb1991) [inline spec/features/merchant_items/index_spec.rb:18]: Really robust test here!

**Review by markcyen — APPROVED** [review]
Looks great! thanks for working on this!

**Pull request merged**